src/components/TodoApp.tsx

```diff
@@ -0,0 +1,21 @@
+import { useContext } from 'react';
+import { TodosContext } from './TodosContext';
+import { Main } from './Main';
+import { Footer } from './Footer';
+import { Header } from './Header';
+
+export const TodoApp: React.FC = () => {
+  const { todos } = useContext(TodosContext);
+
+  return (
+    <>
+      <Header />
+
+      {todos.length !== 0 && (
+        <Main />
+      )}
+
+      <Footer />
```

Comment:
```suggestion
      {todos.length !== 0 && (
        <Main />
        <Footer />
      )}
```
and remove the condition in Footer